modules/ROOT/pages/clauses/listing-procedures.adoc

```diff
@@ -67,12 +67,17 @@ List of roles permitted to use boosted mode when executing this procedure.
 Is `null` without the xref::administration/access-control/dbms-administration.adoc#access-control-dbms-administration-role-management[`SHOW ROLE`] privilege.
 m| LIST OF STRING
 
+m| isDeprecated
+a| Whether the procedure is deprecated.
+m| BOOLEAN
+
 m| option
 a| Map of extra output, e.g. if the procedure is deprecated.
 m| MAP
 
 |===
 
+The deprecation information for procedures is returned both in the `isDeprecated` and `option` columns at this time.
```

Comment:
```suggestion
The deprecation information for procedures is returned both in the `isDeprecated` and `option` columns.
```

Reply:
this was the same as Jens suggestion so I accepted his one 🤷